doc/rados/operations/health-checks.rst

```diff
@@ -209,15 +209,15 @@ The OSD can be removed from the CRUSH hierarchy with::
 OSD_OUT_OF_ORDER_FULL
 _____________________
 
-The utilization thresholds for `backfillfull`, `nearfull`, `full`,
+The utilization thresholds for `nearfull`, `backfillfull`, `full`,
 and/or `failsafe_full` are not ascending.  In particular, we expect
-`backfillfull < nearfull`, `nearfull < full`, and `full <
+`nearfull < backfillfull`, `backfillfull < full`, and `full <
```

Comment:
also, i am wondering if we could revise the way to explain the order of the full ratios. can we just put something like

> In particular, we expect `nearfull` < `backfillfull` < `full` < `failsafe_full`?

Reply:
I think it would be clearer if we do it the way you suggested. Should I fix it?

Reply:
i am fine either way. my only concern is that line-wrap could render the text a little bit difficult to read. but it's a compromise, i guess.

but anyway, it's already an improvement with your change!